test/srt/test_vision_openai_server.py

```diff
@@ -443,6 +443,79 @@ def setUpClass(cls):
     def test_video_chat_completion(self):
         pass
 
+class TestPixtralServer(TestOpenAIVisionServer):
```

Comment:
To save some time on CI, we only need to test one.

Reply:
yea deleted the other one.